packages/ui/css/components/modal-join-path.css

```diff
@@ -0,0 +1,48 @@
+.modal-container {
```

Comment:
This file should not be in `ui`. The join path modal ONLY applies to the `client`

Reply:
I will fix this right away